### DISCUSSION TITLE
Why did you stop working on this?

### DISCUSSION THREAD

**Discussion by joshua-price1** [birdrides/mockingbird#335] [General]
This made swift not a bad language. Please keep updating it!!!

**Comment by legoesbenr** [comment]
@andrewchang-bird @ryanmeisters 

Can you please elaborate on why the development has stopped?
MockingBird seems like a great framework with many advantages, but seeing that releases has been stale for 2+ years and noone responds on the slack channel indicates that the project has died.

What are we to expect? Should we keep looking for other solutions?

**Comment by ryanmeisters** [comment]
Apologies to all for the inactivity and radio silence. This project has had a tough time, and I know it is probably frustrating to many.

I too would like to see this project revived, but don't personally have the time to take it over. And I would assume that that is also the case for Mockingbird's original author.

While I don't have any authority or ownership here anymore, perhaps it would be most productive to start a discussion around transition to community ownership. I'm not sure what that would look like, but happy to help any way that I can.

**Comment by RafaelPena31** [comment]
Maybe this is a good opportunity to fork this project and continue?
